### ISSUE TITLE
Cannot send request properly when schema is uppercase

### ISSUE THREAD

**Issue by konohiroaki** [a85/POSTMan-Chrome-Extension#4085]
Thank you for the great tool. I found a trivial bug so let me report it in this issue ticket. :)

## App Details:
```
Postman for Windows
Version 5.5.0
win32 6.1.7601 / x64
```

## Issue Report:
When using uppercase for schema part, it seems the request is sent wrongly. 

### Example 1 (correct behavior):
Choose `GET` method and set `https://github.com/` in the url form. It sends this request...
```
curl -X GET \
  https://github.com/ \
...
```
The server responds properly.
### Example 2 (wrong behavior?):
Choose  `GET` method and set `HTTPS://github.com/` in the url form. It sends this request...
```
curl -X GET \
  http://HTTPS:/github.com/ \
...
```
and will result with `Could not get any response`.

According to [rfc3986#section-3.1](https://tools.ietf.org/html/rfc3986#section-3.1),
> Although schemes are case-
   insensitive, the canonical form is lowercase and documents that
   specify schemes must do so with lowercase letters.  An implementation
   should accept uppercase letters as equivalent to lowercase in scheme
   names (e.g., allow "HTTP" as well as "http") for the sake of
   robustness but should only produce lowercase scheme names for
   consistency.

**Comment by thecuriousdev** [comment]
@konohiroaki thanks for reporting this issue, we'll look into this.

**Comment by harryi3t** [comment]
This has been fixed in the latest version of the Postman v6.7.1
